packages/docusaurus-theme-translations/locales/pl/theme-common.json

```
@@ -67,6 +67,6 @@
   "theme.tags.tagsListLabel": "Tagi:",
   "theme.tags.tagsPageLink": "Wyświetl wszystkie tagi",
   "theme.tags.tagsPageTitle": "Tagi",
-  "theme.unlistedContent.message": "This page is unlisted. Search engines will not index it, and only users having a direct link can access it.",
-  "theme.unlistedContent.title": "Unlisted page"
+  "theme.unlistedContent.message": "Ta strona jest niepubliczna. Wyszukiwarki nie będą go indeksować, a dostęp do niego będą mieli tylko użytkownicy posiadający bezpośredni link.",
```

```suggestion
  "theme.unlistedContent.message": "Ta strona jest niepubliczna. Wyszukiwarki nie będą jej indeksować, a dostęp do niej będą mieli tylko użytkownicy posiadający bezpośredni link.",
```

Possessive pronoun correction.